### DISCUSSION TITLE
Podcasts only pulls in 3 episodes

### DISCUSSION THREAD

**Discussion by ahanson24** [advplyr/audiobookshelf#797] [Q&A] [answered]
I'm trying to pull in an entire podcast. When I go to edit and update "Look for new episodes after this date" to 1 year ago, it always only pulls in 3 episodes. Am I missing a setting or something?

**Comment by ahanson24** [comment, marked as answer]
I found the solution. Just needed to click Find Episodes I can select all.

**Comment by gbakeman** [comment]
I think this is a good opportunity for an improvement, as the blank initial episode list was confusing to me too. I think feed metadata should atleast be parsed and displayed, then the user can choose to stream or download.
